py4DSTEM/process/diffraction/crystal_calibrate.py

```diff
@@ -0,0 +1,465 @@
+
+import numpy as np
+import matplotlib.pyplot as plt
+from fractions import Fraction
+from typing import Union, Optional
+from copy import deepcopy
```

Comment:
i think using deepcopy is a little dangerous, and should no longer be needed since the BraggVectors.copy() function has been updated